games/datingsim/master.py

```diff
@@ -752,16 +789,38 @@ def compute_scores(self, episode_interactions: Dict) -> None:
                     turn_score["out_of_reprompts"] = 1
                     aborted = True
 
-                if action["type"] == "agreement": # agreement rate
+                if action["type"] == "overall agreement": # success
                     turn_score["last_message"] = action["content"]
-                    turn_score["agreement"] = 1
+                    turn_score["success"] = 1
 
                 if action["type"] == "friendzone": # mismatch_agreement
                     turn_score["last_message"] = action["content"]
-                    turn_score["agreement"] = 0
+                    turn_score["success"] = 0
 
                 if action["type"] == "out of turns": # no agreement settled in max amount of turns
-                    turn_score["agreement"] = 0
+                    turn_score["success"] = 0
+                    turn_score["out of turns"] = 1
+                
+                if action["type"] == "time agreement": #action/location/time
+                    turn_score["last_message"] = action["content"]
+                    turn_score["time agreement"] = 1
+                
+                if action["type"] == "action agreement": #action/location/time
+                    turn_score["last_message"] = action["content"]
+                    turn_score["action agreement"] = 1
+                
+                if action["type"] == "location agreement": #action/location/time
+                    turn_score["last_message"] = action["content"]
+                    turn_score["location agreement"] += 1
```

Comment:
> in the previous two blocks of code ("action agreement" and "time agreement") we have turn_score["something"] = 1 and here we have +=1. Is this intentional?

not intentional, just a typo. It would not cause any problem as it processes one single turn each time but lemme fix it =)